.github/workflows/job_ci-checks.yml

```
@@ -0,0 +1,43 @@
+name: CI Checks
+
+on:
+  workflow_call:
+    inputs:
+      WORKING_DIRECTORY:
+        description: Path to working directory in repo
+        required: true
+        type: string
+
+jobs:
+  ci-checks:
+    name: Run CI checks
+    runs-on: ubuntu-24.04
```

suggestion: Disable sam telemetry for all steps in this job 

https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-telemetry.html



```suggestion
    runs-on: ubuntu-24.04
    env:
      SAM_CLI_TELEMETRY: 0
```

---

Is this because we don't want data gathered even if it's not pii?

---

Yeah, hands off our data! Also it slows down the steps by a second and outputs log lines we don't like!